README.md

```diff
@@ -1,40 +1,46 @@
-FusionAuth Device Authorization Grant workflow example
-====
-This code base, was orginally built on top of the FusionAuth main Device Authorization Grant flow example, you can find here the original example on the following [github repository] (https://github.com/FusionAuth/fusionauth-example-device-grant)
+# FusionAuth Device Authorization Grant Workflow Example
 
-[Read the full article](https://fusionauth.io/learn/expert-advice/gaming-entertainment/oauth-device-grant-gaming) walking you through the Device Authorization grant and how it can help your users log in to your game.
+This codebase is built on top of the FusionAuth main Device Authorization Grant flow example. You can find the original example on the [FusionAuth GitHub repository](https://github.com/FusionAuth/fusionauth-example-device-grant).
 
-This is for demo puposes
-====
-A basic HTML/JavaScript app demonstrating the Device Authorization Request and Device Access Token Request of the [OAuth2 Device Authorization Grant](https://tools.ietf.org/html/rfc8628) specification.
+Read the full article walking you through the Device Authorization grant and how it can help your users log in to your game.
```

Comment:
```suggestion
Read the full article walking you through the Device Authorization Grant and how it can help your users log in to your application.
```